modules/src/webhook-spawn.sh

```
@@ -425,16 +436,88 @@ fi
+claim_note() {
+  # The note is echoed under every delivery, so it has to say WHAT this
+  # session was spawned for — a fresh-context session reads it to know why the
+  # event matters. It used to be one fixed sentence, byte-identical across
+  # every hook session on the box (issue #251).
+  "$JQ" -rn --argjson meta "${LOCAL_WEBHOOK_SPAWN_META:-{\}}" --arg key "$1" '
+    ($meta.number // "" | tostring) as $n
+    | ($meta.event // "") as $e
+    | ($meta.action // "") as $a
+    | (if $e == "" then "" else " (" + $e + (if $a == "" then "" else "." + $a end) + ")" end) as $what
+    | if $n != "" then
+        "seeded at spawn: this session was started for \($key)#\($n)\($what) and claims that object'"'"'s events while it lives"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)